zebra-state/src/service/memory_state.rs

```diff
@@ -460,11 +402,199 @@ impl Ord for Chain {
     }
 }
 
+/// The state of the chains in memory, incuding queued blocks.
+#[derive(Default)]
+pub struct NonFinalizedState {
+    /// Verified, non-finalized chains.
```

Comment:
`BTreeSet`s are in ascending order.

```suggestion
    /// Verified, non-finalized chains, in ascending order.
    ///
    /// The best chain is `chain_set.last()` or `chain_set.iter().next_back()`.
```